packages/monocdk/package.json

```diff
@@ -345,6 +345,7 @@
     "@aws-cdk/lambda-layer-node-proxy-agent": "0.0.0",
     "@aws-cdk/pipelines": "0.0.0",
     "@aws-cdk/pkglint": "0.0.0",
+    "@aws-cdk/triggers": "0.0.0",
```

Comment:
Can we put this alphabetically below `"@aws-cdk/region-info": "0.0.0",`, like we do in `aws-cdk-lib/package.json`?